sources/server/token_handler.cpp

```diff
@@ -173,38 +173,39 @@ std::string
 serv::TokenHandler::doAction(const Command& aCommand) noexcept
 {
     std::string res = "No token command applied.";
-    if (aCommand.argument == "turn_off")
+    std::map<std::string, std::function<std::string()>> tokens_mode = 
     {
-        res = "Tokens turned OFF!";
-        core::VariableStorage::setVariable("token:isActive", false);
-        core::VariableStorage::setVariable("token:isMemory", false);
-    }
-    else if (aCommand.argument == "turn_on")
-    {
-        res = "Tokens turned ON!";
-        core::VariableStorage::setVariable("token:isActive", true);
-        core::VariableStorage::setVariable("token:isMemory", false);
-    }
-    else if (aCommand.argument == "memory")
-    {
-        res = "Start url-role memorization.";
-        core::VariableStorage::setVariable("token:isActive", false);
-        core::VariableStorage::setVariable("token:isMemory", true);
-    }
-    else if (aCommand.argument == "print")
-    {
-        res = "Print url-role data.";
-        printAutorisation();
-    }
-    else if (aCommand.argument == "cut")
-    {
-        core::VariableStorage::setVariable("insert_tokens", false);
-    }
-    else if (aCommand.argument == "add")
-    {
-        core::VariableStorage::setVariable("insert_tokens", true);
+        {"turn_off", [](){
```

Comment:
I don't understand why we need lumda here. As for me, I was thinking about the switch-case operator. It would also be great if you could find a better way to generate the users accessibility rules.